.github/workflows/qa.yaml

```diff
@@ -0,0 +1,32 @@
+name: "UI deploy QA"
+on:
+  release:
```

Comment:
As we want this on merge to master it should be
```
on:
  push:
    branches:
      - master
```

Reply:
Done